test/quota-test.js

```diff
@@ -0,0 +1,93 @@
+const quota = require('../quota/index');
+const assert = require('assert');
+
+var exampleConfig = { 
+  EdgeMicroTestProduct: {
+    allow: '3',
+    interval: '1',
+    timeUnit: 'minute',
+    bufferSize: 10000,
+    uri: 'https://edgemicroservices-us-east-1.apigee.net/edgemicro/quotas/organization/ws-poc3/environment/test',
+    key: 'b7b69b6bd79e778782af9a653ae9c4d91e65440d8acbc7e969437a8a8dc13612',
```

Comment:
not sure we want these hard coded in the tests, for security reasons.  What do you think?